cli/js/web/url.ts

```diff
@@ -389,7 +389,7 @@ export class URLImpl implements URL {
         hash: urlParts.hash,
       });
     } else {
-      throw new TypeError("URL requires a base URL.");
+      throw new TypeError("Invalid URL.");
```

Comment:
Not sure about changing this one, but I thought this was more inline with other implementations.